common/beerocks/bwl/dummy/base_wlan_hal_dummy.cpp

```diff
@@ -14,9 +14,12 @@
 #include <beerocks/bcl/son/son_wireless_utils.h>
 
 #include <easylogging++.h>
+#include <limits.h>
 #include <sys/inotify.h>
 
 #define UNHANDLED_EVENTS_LOGS 20
+#define EVENT_SIZE (sizeof(struct inotify_event))
+#define EVENT_BUF_LEN (1024 * (EVENT_SIZE + NAME_MAX + 1))
```

Comment:
Could you please explain the calculation?
I see in the man page the reason for `sizeof(struct inotify_event) + NAME_MAX + 1` but couldn't understand why did you multiply it by 1024.
It is better to add a comment explaining it, even what said on the man page.